src/hevm/src/EVM.hs

```diff
@@ -81,10 +81,7 @@ data Error
   | MaxCodeSizeExceeded Word Word
   | PrecompileFailure
 
-instance Show Error
-  where
-    show (Revert msg) = "Revert " ++ showByteStringWith0x msg
```

Comment:
I think we need this so that revert messages are rendered in hexadecimal, rather than literally?

Reply:
Ah, ok. I reverted to the derived version becasue this was causing an infinite loop. I'll reinstate a working version.